src/lib/cache/subCache.cpp

```diff
@@ -705,20 +707,25 @@ void subCacheItemInsert
   //
   // 1. First the non-complex values
   //
-  cSubP->tenant                = (tenant[0] == 0)? NULL : strdup(tenant);
-  cSubP->servicePath           = strdup(servicePath);
-  cSubP->subscriptionId        = strdup(subscriptionId);
-  cSubP->reference             = strdup(scrP->reference.get().c_str());
-  cSubP->expirationTime        = expirationTime;
-  cSubP->throttling            = throttling;
-  cSubP->lastNotificationTime  = lastNotificationTime;
-  cSubP->notifyFormat          = notifyFormat;
-  cSubP->next                  = NULL;
-  cSubP->count                 = (notificationDone == true)? 1 : 0;
-  cSubP->expression.q          = q;
-  cSubP->expression.geometry   = geometry;
-  cSubP->expression.coords     = coords;
-  cSubP->expression.georel     = georel;
+  cSubP->tenant                  = (tenant[0] == 0)? NULL : strdup(tenant);
```

Comment:
Yes, apart for the adding of if (stringFilterP != NULL) ... further down.
To tell you the truth I don't know why the indentation has been changed.
Probably at some time I had another assignment in the list, that required the new indentation.
Yes!   I had 

    cSubP->expression.stringFilter = *stringFilterP;

But as often stringFilterP is NULL, I got hundreds of sigsegvs in functests due to that line ... :-)